docker/local-universe/README.md

```diff
@@ -75,6 +75,52 @@
     ```bash
     $ sudo make local-universe
     ```
+
+## Building Your Own, off a non-changing universe-static base image.
+
+Mesosphere provides a `mesosphere/universe-static` Docker image, which has all of the core requirements to run a local universe.  All that must be added are your own certificates and repo contents.
+
+1. Use the make command to download the static image
+
+    ```bash
+    $ sudo make static-online
+    ## Will pull and re-tag mesosphere/universe-static:0.2 to universe-static
+    ```
+
+1. Create your certs (either use `make certs` or generate your own), and add them to the static image to create a `universe-base` image
+
+    ```bash
+    $ sudo make static-base
+    ## Will add certs to universe-static and create universe-base
+    ```
+
+1. Add content (see above)
+
+    ```bash
+    $ sudo make local-universe
+    ```
+
+To generate your own static image, use and/or modify the make target `static-build` (e.g., `make static-build` instead of `make static-online`)
+
+This leaves three total options:
+
+```bash
+make base
+make local-universe
```

Comment:
I have no idea what it takes to get our CI to do that - any suggestions?  Also, I think the idea is that we shouldn't be rebuilding mesosphere/universe-static:0.2 (the hash should never ever change).  

Re docs: for now, we (field team) are working with all of our customers who actually use this, so we can walk them through the process.  I'll work on getting docs updated, although I've heard that can be difficult.